src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -317,6 +318,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
       lv_crossout = zif_abapgit_html=>c_html_opt-crossout.
     ENDIF.
     ro_advanced_dropdown->add( iv_txt = 'Uninstall'
+                               iv_title = `Deletes SAP package objects!`
```

Comment:
```suggestion
                               iv_title = `Delete all objects in this package from the system`
```

Reply:
... and of all subpackages

Reply:
... depending on the ignore subpackages setting

Reply:
In line with the above, what do you think of:
```suggestion
                               iv_title = `Delete all development objects belong to this package (and subpackages) from the system!`
```

Both this and the "remove" suggestion above, seem to fit:
![image](https://user-images.githubusercontent.com/34446306/99937051-5a9f3300-2d6d-11eb-9d2b-eafbb39f05c9.png)

Reply:
i don't like exclamation marks in documentation, but maybe it's just a personal preference

Reply:
+1 to avoid exclamation marks!

Reply:
Copy that, I don't like them either frankly. I thought an exception was warranted, to caution people as uninstall is arguably "dangerous". But I suppose the confirmation dialog which follows covers the concern. 
So:
```suggestion
                               iv_title = `Delete all development objects belonging to this package (and subpackages) from the system`
```